ios and android ?

I gave it a shot anyway to see ....

IOS:

```  
gogio -o godcr.app -target ios .
xcrun simctl install booted godcr.app 
``` 

![Screenshot 2022-04-22 at 12 19 43](https://user-images.githubusercontent.com/53147028/164689478-1b04b704-cd55-4e96-9b46-f246f93ee00b.png)


ANDROID:

```
gogio -target android .
gogio: go build -ldflags=-w -s -X gioui.org/app/internal/log.appID=com.github.godcr -buildmode=c-shared -tags  -o /var/folders/wp/ff6sz9qs6g71jnm12nj2kbyw0000gp/T/gogio-3048283512/jni/x86/libgio.so . failed: # github.com/lib/pq
../../../../pkg/mod/github.com/lib/pq@v1.10.3/user_posix.go:13:6: userCurrent redeclared in this block
	../../../../pkg/mod/github.com/lib/pq@v1.10.3/user_other.go:8:6: other declaration of userCurrent

```

has a postres driver dependency that is messing things up for android ?

--- a/go.mod
+++ b/go.mod
@@ -24,3 +24,9 @@ require (
 	golang.org/x/image v0.0.0-20210628002857-a66eb6448b8d
 	golang.org/x/text v0.3.7
 )
+
+// Older versions of github.com/lib/pq are required by politeia (v1.9.0)
+// and dcrdex (v1.10.3) but only v1.10.4 and above can be compiled for
+// the android OS using gomobile. This replace can be removed once any
+// of those projects update their github.com/lib/pq dependency.
+replace github.com/lib/pq => github.com/lib/pq v1.10.4

